skyportal/handlers/api/spectrum.py

```diff
@@ -587,3 +588,33 @@ def get(self, obj_id):
                 )
 
         return self.success(data={'obj_id': obj.id, 'spectra': return_values})
+
+
+class SpectrumRangeHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        """Docstring appears below as an f-string."""
```

Comment:
Still needs docstring, preferably inline right here as opposed to adding it below

Reply:
I've added a docstring, but please check the format.